package.nls.json

```diff
@@ -6,6 +6,7 @@
     "extension.commands.salesforce-mobile-offline.lwc-mobile.version": "Version of ESLint Plugin LWC Mobile to include in devDependencies",
     "extension.commands.salesforce-mobile-offline.komaci.version": "Version of ESLint Plugin LWC Graph Analyzer to include in devDependencies",
     "extension.commands.salesforce-mobile-offline.eslint.version": "Version of ESLint to include in devDependencies",
+    "extension.commands.salesforce-mobile-offline.toolinghub.liveKomaciAnalysis": "(TODO) Salesforce Mobile Offline: Live Komaci Analysis",
```

Comment:
Nit: This is where you make use of the `category` configuration option in `package.json`, similar to the [linting command](https://github.com/salesforce/salesforcedx-vscode-mobile/blob/e053630226add27c9d75a8c4599306625f95c63e/package.json#L55). So you'd take out the "Salesforce Mobile Offline:" part here, and let the category value/configuration add the prefix.

Reply:
I had that originally actually - but while developing, I noticed it doesn't append the category consistently? I forget exactly which one out of command palette, editor, and context, but one or two of them was missing category.

I figure this is why the base project opts to have "SFDX:" coded into the command label itself.
https://github.com/forcedotcom/salesforcedx-vscode/blob/232e5af3d5d18d4816338e891175ade9bc3f37bf/packages/salesforcedx-vscode-core/package.nls.json